packages/react-ui/internal/ThemePlayground/__stories__/ThemePlayground.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { CSFStory } from 'creevey';
+
+import { ThemeContextPlayground } from '../../../internal/ThemePlayground/ThemeContextPlayground';

```

Comment:
Путь слишком длинный.
```suggestion
import { ThemeContextPlayground } from '../ThemeContextPlayground';
```